### ISSUE TITLE
Cannot change the menuitem programmatically!

### ISSUE THREAD

**Issue by elar-systems** [roughike/BottomBar#401]
I have a page with two fragments which using the ViewPager to switch between fragments.
I can use this BottonBar to navigate between fragments by changing the "myviewpager.setCurrentItem(int)" and it works perfectly.

However, when I change the fragments by swiping the finger left or right, the menuitem in BottonBar need to be adjusted and set to the correct menuitemid.

How can I do that programmatically?
I thought "mBottonBar.setDefaultTabPosition(int)" might be able to do that, but how can I find my BottonBar in any of the Fragments to be able to set it? (Cannot find it by ID!!!)

**In short:** 
1- Can we switch between BottonBar Tabs programmatically when ViewPagerTab changes the fragments? (Resolved, see my comments)
2- In above scenario, can I have to different BottonBar for each of the Fragments?

**Comment by elar-systems** [comment]
Guys, 
Question #1 is resolved by adding the "addOnPageChangeListener" for my ViewPager and  using the "onPageSelected" method.
I then used "mBottonBar.setDefaultTabPosition(int)" to assign the correct tab base on my ViewPager position.

I still need to know if I can add to different BottonBar to my different Fragments in my ViewPage Activity.

Many Thanks

**Comment by roughike** [comment]
You can use ```selectTabWithId()``` and ```selectTabAtPosition()``` for programmatically selecting tabs.

Don't add multiple BottomBars to separate Fragments: use one in an Activity that contains your Fragments instead.
